packages/twenty-server/src/modules/match-participant/utils/build-person-email-query-builder.util.ts

```diff
@@ -0,0 +1,51 @@
+import { SelectQueryBuilder } from 'typeorm';
+
+import { PersonWorkspaceEntity } from 'src/modules/person/standard-objects/person.workspace-entity';
+
+export interface BuildPersonEmailQueryBuilderOptions {
+  queryBuilder: SelectQueryBuilder<PersonWorkspaceEntity>;
+  emails: string[];
+  excludePersonIds?: string[];
+}
+
+export function buildPersonEmailQueryBuilder({
```

Comment:
sounds to me that you forgot the "withDeleted: true" no ? unless it is by default using the query builder